lib/tasks/email_missed_reservations.rake

```diff
@@ -0,0 +1,16 @@
+desc 'Send emails about missed reservations'
+task email_missed_reservations: :environment do
+  # get all reservations that are approved and missed, send an email to
+  # user to inform them
+  missed_reservations = Reservation.where(
```

Comment:
We need to check our application setting to determine whether or not to run this task (see the old [line 12](https://github.com/YaleSTC/reservations/pull/1115/files#diff-725442a5e753d0e83aa1d2844543c74eL12) above).